src/main/java/com/sarality/db/query/Query.java

```diff
@@ -15,10 +15,21 @@
   // Order by clause for the query
   private final String orderByClause;
 
+  //Group By Clause
+  private final String groupByClause;
+  private final String[] selectColumns;
+
   public Query(String whereClause, String[] whereClauseArguments, String orderByClause) {
+    this(new String[] {}, whereClause, whereClauseArguments, orderByClause, null);
+  }
+
+  public Query(String[] selectColumns, String whereClause, String[] whereClauseArguments, String orderByClause,
```

Comment:
Might as well add a constructor with having and limit as well and make this complete